docs/usage/config-presets.md

```diff
@@ -119,39 +119,9 @@ Or if you think your preset would be valuable for others, please contribute a PR
 If you manage multiple repositories using Renovate and want the same custom config across all or most of them, then you might want to consider publishing your own preset config so that you can "extend" it in every applicable repository.
 That way when you want to change your Renovate configuration you can make the change in one location rather than having to copy/paste it to every repository individually.
 
-Let's say that your username on npm and elsewhere is "fastcore".
-In that case, you can choose between publishing your preset config package as `@fastcore/renovate-config` or `renovate-config-fastcore`.
-Let's assume you choose `renovate-config-fastcore` as the package name:
+## File based presets example
 
-You then need to publish the `renovate-config-fastcore` package where the `package.json` contains the field `renovate-config` and then put your config under the field `default`.
-For example:
-
-```json
-{
-  "name": "renovate-config-fastcore",
-  "version": "0.0.1",
-  ...
-  "renovate-config": {
-    "default": {
-      "extends": ["config:base", "schedule:nonOfficeHours"]
-    }
-  }
-}
-```
-
-Then in each of your repositories you can add your renovate config like:
-
-```json
-  "extends": ["fastcore"]
-```
-
-Any repository including this config will then adopt the rules of the default `library` preset but schedule it on weeknights or weekends.
-
-Note: if you prefer to publish using the namespace `@fastcore/renovate-config` then you would use the `@` prefix instead:
-
-```json
-  "extends": ["@fastcore"]
-```
+<!-- TODO: Explain how to use a file based preset here. -->
```

Comment:
Do we have file-based presets?

Reply:
@viceice mentioned `file-based presets` in here: https://github.com/renovatebot/renovate/issues/8464#issue-796948938

Reply:
I mean the `local`, `github`, `gitlab` .... hosted presets 🙃

Reply:
@viceice Can you maybe push a commit that gives me an idea of what changes you want? I'm really confused on what we actually want to change here, sorry!

Reply:
OK, i checked the published docs now. Maybe we should simply rename `How to Publish Preset Configs` to ` NPM hosted presets` and move to last on the page.

🤔  I'm really not sure too about what to write here.

Reply:
I've removed the `file based preset` thing completely.
I've renamed the `How to publish preset configs` to `npm-hosted presets` and moved it to the bottom of the page.

Is this what we want?

Reply:
@rarkins and @viceice Can you take a look at this again, and tell me what you think? I'm not sure what we need to get this merged?